concent_api/core/message_handlers.py

```diff
@@ -1117,7 +1125,7 @@ def update_subtask(
     )
 
     # Concent should send e-mail notification when the last active subtask switches to a passive state.
-    if config.SOFT_SHUTDOWN_MODE is True and not Subtask.objects.filter(state__in=Subtask.ACTIVE_STATES).exists():
+    if config.SOFT_SHUTDOWN_MODE is True and not Subtask.objects.filter(state__in=Subtask.ACTIVE_STATES).exists():  # pylint: disable=no-member
```

Comment:
You did not even change this line. Why is the ignore necessary? Did someone leave an unfixed warning?

Please make sure you're running the latest version of pylint.

Reply:
@MichalW-CP What about my question here?

Reply:
Pylint version is up to date